doc/nrf/device_guides/nrf70/nrf7002eb_54l15pdk_gs.rst

```diff
@@ -0,0 +1,79 @@
+:orphan:
+
+.. _ug_nrf7002eb_nrf54l15pdk_gs:
+
+Getting started with nRF7002 EB for nRF54L15 PDK
+################################################
+
+.. contents::
+   :local:
+   :depth: 4
+
+This section builds on :ref:`nRF7002 EB <ug_nrf7002eb_gs>` documentation, only changes specific to :ref:`zephyr:nrf54l15pdk_nrf54l15` are mentioned here.
+
+Overview
+********
+
+The nRF7002 EB is a shield that can be used with the nRF54L15 PDK to add support for Wi-Fi.
+
+Limitations
+===========
+
+The Wi-Fi support is experimental and has the following limitations:
+
+* It only supports STA mode
+* It only allows Wi-Fi open security
+* It is only suitable for low throughput applications
+
+Pin mapping for nRF54L15PDK
+###########################
+
+The pin mapping for the nRF54L15 PDK is as follows:
+
++-----------------------+-------------------+-----------------------------------------------+
+| nRF700x pin name (EB) | nRF54L15 PDK pins | Function                                      |
```

Comment:
```suggestion
| nRF70 Series pin name (EB) | nRF54L15 PDK pins | Function                                      |
```

Reply:
@bama-nordic and @busineni has specifically suggested this change, so, will leave it to them to take a final call.

Reply:
OK, our basis was this https://nordicsemi.atlassian.net/browse/SHEL-1853. But if it can stay like this, its fine.

Can we write **nRF7002 EB signal** like it is written here for EK: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_nrf7002ek.html?

Reply:
Sure, that makes sense.

@bama-nordic @busineni please confirm if you are okay with these recommendations.

Reply:
Okay, just FYI "nRF700x" is not allowed according to Marketing: https://nordicsemi.atlassian.net/wiki/spaces/MAR/pages/132940950/Product+names